packages/code-block/src/insertFragmentCodeBlock.ts

```diff
@@ -30,21 +30,25 @@ export const insertFragmentCodeBlock = <V extends Value>(
   }
 
   return (fragment: TDescendant[]) => {
-    const inCodeLine = findNode(editor, { match: { type: codeLineType } });
-    if (!inCodeLine) {
-      return insertFragment(fragment);
+    const aboveBlockNode = getBlockAbove(editor)?.at(0) as TElement;
```

Comment:
Thanks for the PR! Does this work instead?

```suggestion
    const [blockAbove] = getBlockAbove<TElement>(editor) ?? [];
```

Reply:
Thanks for the suggestion, works as expected. I'll fix it